ui/datasets/semantic-ui/js/datasets/citations/components/RecordCitations.jsx

```diff
@@ -4,7 +4,18 @@ import PropTypes from "prop-types";
 import { i18next } from "@translations/i18next";
 import { Loader, Segment } from "semantic-ui-react";
 
-const CitationList = lazy(() => import("./CitationList"));
+const CitationField = lazy(() => import("./CitationField"));
+
+const CITATION_STYLES = [
```

Comment:
These 2 constants need to come from backend config (what styles are supported there can change)

Reply:
OK, we'll solve as part of common backend integration/implementation in a different issue :)